src/nvim/path.c

```diff
@@ -1553,21 +1552,19 @@ int path_with_url(char_u *fname)
  */
 int vim_isAbsName(char_u *name)
```

Comment:
I suppose I should clarify: the only reason i mentioned it was because of how easy it would be to convert. All call sites already expect a boolean result, so only the return type would need to be changed.

Still, I understand if you don't want to as your PR already covers a lot.